plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -222,6 +225,8 @@ export class PgInstrumentation extends InstrumentationBase<PgInstrumentationConf
   }
 
   private recordOperationDuration(attributes: Attributes, startTime: HrTime) {
+    const emit = utils.getEmitType();
+    if (emit === utils.emitType.OLD_SEM_CONV) return;
```

Comment:
With this the semconv stability env-var this actually becomes a breaking change. I'd recommend splitting it form this PR and only introducing the bugfix first - then we can still follow up with the stability opt-in feature that was requested in https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2479#issuecomment-2415615704

Reply:
sounds good, working on it

Reply:
done